infrastructure/app-service.tf

```diff
@@ -0,0 +1,37 @@
+resource "azurerm_app_service_plan" "main" {
+  name                = "terraformed-asp"
+  location            = data.azurerm_resource_group.main.location
+  resource_group_name = data.azurerm_resource_group.main.name
+  kind                = "Linux"
+  reserved            = true
+
+  sku {
+    tier = "Basic"
+    size = "B1"
+  }
+}
+
+resource "azurerm_app_service" "main" {
+  name                = "AlexKApp-${random_string.random_identifier.result}"
```

Comment:
Why use a random identifier? Are you expecting to spin up an arbitrary number of environments? You'd usually want them to be called e.g. -prod, -dev, -uat rather than a random string

Reply:
Think this is just force of habit from working on gcp for too long where most things need to be globably unique and best practice is to include a random string.